src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int number = random.nextInt(BOUND);
+        String color = colorSupplier.getRandomColor().name();
```

Comment:
you need to call `toLowerCase()` as well

`name()` will return you `WHITE` instead of `white`

```suggestion
        String color = colorSupplier.getRandomColor().name().toLowerCase();
```